app/global-styles.js

```diff
@@ -13,6 +13,9 @@ import { singleCommunityStyles } from './utils/communityManagement';
 
 const styles = singleCommunityStyles();
 
+export const italicFont = () =>
```

Comment:
There is not a reason to call this function several times. It should be enough to execute once and save the value in a global variable.